src/api/telemetry/TelemetryMetadataManager.js

```diff
@@ -78,6 +76,30 @@ define([
         valueMetadata.source = valueMetadata.source || valueMetadata.key;
         valueMetadata.hints = valueMetadata.hints || {};
 
+        if (valueMetadata.hints.hasOwnProperty('x')) {
+            console.warn(
+                'DEPRECIATION WARNING: `x` hints should be replaced with ' +
+                '`domain` hints moving forward.  ' +
+                'https://github.com/nasa/openmct/issues/1546'
+            );
+            if (!valueMetadata.hints.hasOwnProperty('domain')) {
+                valueMetadata.hints.domain = valueMetadata.hints.x;
+            }
+            delete valueMetadata.hints.x;
+        }
+
+        if (valueMetadata.hints.hasOwnProperty('y')) {
+            console.warn(
+                'DEPRECIATION WARNING: `y` hints should be replaced with ' +
+                '`range` hints moving forward.  ' +
+                'https://github.com/nasa/openmct/issues/1546'
+            );
+            if (!valueMetadata.hints.hasOwnProperty('range')) {
+                valueMetadata.hints.range = valueMetadata.hints.y;
+            }
+            delete valueMetadata.hints.y;
```

Comment:
`delete` is generally slow, so can we just set `valueMetadata.hints.y` to `null` and when the garbage collector will run, it will free up the memory.